v3/src/models/data/attribute.ts

```diff
@@ -313,6 +313,31 @@ export const Attribute = types.model("Attribute", {
     }
   }
 }))
+.views(self => ({
+  get toArchive() {
```

Comment:
Creating a new model was causing a MST error about duplicate ids. I just created the return object directly in the attribute handler, which works and keeps this CODAP specific code out of the attribute model. This might be trickier to do when it comes to updating, though, since we'll want to update everything in a single action that's `withoutUndo`.

Reply:
I wouldn't have thought that creating a local `SnapshotProcessor` would trigger such a warning, but I didn't try it locally. 🤔